.github/workflows/release.yml

```diff
@@ -32,8 +32,10 @@ jobs:
                   ref: gh-pages
 
             - name: 🔪 Prepare
+              env:
+                  GITHUB_REF_NAME: ${{ github.ref_name }}
               run: |
-                  tag="${{ github.ref_name }}"
+                  tag="${GITHUB_REF_NAME}"
```

Comment:
This var is now redundant, can be accessed directly in L39 to strip the leading `v`